packages/client/components/TopBarAvatar.tsx

```diff
@@ -16,7 +16,7 @@ const AvatarWrapper = styled('button')({
   borderRadius: 100,
   marginLeft: 8,
   padding: 4,
-  ':focus': {
+  ':focus-visible': {
```

Comment:
i hate when the accessibility blue outline shows up when you click! it should show up when you tab, not click.